stanford.profile

```diff
@@ -133,6 +132,13 @@ function stanford_sites_tasks() {
   }
   else {
     module_enable(array('webauth'));
+    if(stanford_sites_hosted()) {
+      stanford_sites_add_webauth_user(
+        variable_get('stanford_sites_requester_sunetid'),
```

Comment:
If they are not set, variable_get() defaults to NULL. Then, stanford_sites_add_webauth_user() has error handling in that case.

It probably wouldn't hurt to add a default to these variable_get() calls, but it wouldn't change any functionality.